diff --git a/KVS/kvs.conf b/KVS/kvs.conf
index fb0a7636..a2f32dc3 100644
--- a/KVS/kvs.conf
+++ b/KVS/kvs.conf
@@ -2,19 +2,19 @@
 #  KVS enable/support flags
 #  $Id: kvs.conf 1525 2013-04-11 02:58:08Z naohisa.sakamoto@gmail.com $
 #=============================================================================
-KVS_ENABLE_OPENGL                = 1
-KVS_ENABLE_GLU                   = 1
-KVS_ENABLE_GLEW                  = 1
+KVS_ENABLE_OPENGL                = 0
+KVS_ENABLE_GLU                   = 0
+KVS_ENABLE_GLEW                  = 0
 KVS_ENABLE_OPENMP                = 0
 KVS_ENABLE_DEPRECATED            = 0

 KVS_SUPPORT_CUDA                 = 0
-KVS_SUPPORT_GLUT                 = 1
+KVS_SUPPORT_GLUT                 = 0
 KVS_SUPPORT_GLFW                 = 0
 KVS_SUPPORT_FFMPEG               = 0
 KVS_SUPPORT_OPENCV               = 0
-KVS_SUPPORT_QT                   = 1
-KVS_SUPPORT_OPENXR               = 1
+KVS_SUPPORT_QT                   = 0
+KVS_SUPPORT_OPENXR               = 0
 KVS_SUPPORT_PYTHON               = 0
 KVS_SUPPORT_MPI                  = 0
 KVS_SUPPORT_EGL                  = 0
