diff --git a/Server/arch/Makefile_machine_gcc_omp b/Server/arch/Makefile_machine_gcc_omp
index 6ab1615d..5e3880d5 100644
--- a/Server/arch/Makefile_machine_gcc_omp
+++ b/Server/arch/Makefile_machine_gcc_omp
@@ -3,4 +3,4 @@ CXXFLAGS=-O3 -fopenmp -march=native -std=c++17
 CC=gcc
 CCFLAGS=-O3 -fopenmp -march=native -fcommon
 LD=g++
-LDFLAGS=-fopenmp -lstdc++ -static-libgcc -static-libstdc++
+LDFLAGS=-fopenmp -lstdc++
